Tests/Engine/GetScriptAnalyzerRule.tests.ps1

```diff
@@ -61,16 +61,16 @@ Describe "Test Name parameters" {
 
         It "get Rules with no parameters supplied" {
 			$defaultRules = Get-ScriptAnalyzerRule
-            $expectedNumRules = 43
+            $expectedNumRules = 45
             if ((Test-PSEditionCoreClr))
             {
-                $expectedNumRules = 42
+                $expectedNumRules = 44
             }
 			$defaultRules.Count | Should be $expectedNumRules
 		}
 
         It "is a positional parameter" {
-            $rules = Get-ScriptAnalyzerRule *alias*
+            $rules = Get-ScriptAnalyzerRule "PSAvoidUsingCmdletAliases"
```

Comment:
I considered just updating the count. However, given that this is a test for the positional parameter and not necessarily the use of regex matching, the change I used will prevent this test from failing for similar rule additions in the future. I can swap it to your suggestion if you feel that it is better that way however.